mne_bids/io.py

```diff
@@ -63,7 +63,7 @@ def _read_raw(raw_fname, electrode=None, hsp=None, hpi=None, config=None,
                               preload=False, verbose=verbose)
 
     elif ext in ['.fif', '.ds', '.vhdr', '.set']:
-        raw = reader[ext](raw_fname)
+        raw = reader[ext](raw_fname, allow_maxshield=allow_maxshield)
```

Comment:
ufff looked like I merged this PR hastily. The option `allow_maxshield` does not apply to all readers, only for fif readers. @dnacombo can you send a PR to fix this? Thanks